DayrSuryan/DSEthiop1.xml

```diff
@@ -590,7 +590,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                           <!-- same hand as for other DS MSS?-->
                            <q xml:lang="gez">
                               <hi rend="rubric">ዝመጽሐፍ፡ ዘ<persName ref="PRS11905GabraM" role="owner"><roleName type="title">አቡነ፡</roleName> ገብረ፡ ማርያም፡</persName>
-                               <sic>ዘብዘብሔረ፡</sic> <placeName ref="LOC3577Gondar">ጎንደር፡</placeName>
```

Comment:
this is already marked up with sic (and I could check the images): it is like this in the manuscript